openfl/interface/aggregator.py

```diff
@@ -90,6 +91,21 @@ def start_(plan, authorized_cols, task_group):
             parsed_plan.config["assigner"]["settings"] = {}
         parsed_plan.config["assigner"]["settings"]["selected_task_group"] = task_group
         logger.info(f"Setting aggregator to assign: {task_group} task_group")
+    
+    # Check if remote attestation is enabled in the plan configuration
+    if parsed_plan.config["network"]["settings"].get("enable_remote_attestation", False):
+        # Fetch remote attestation environment variables
+        env_vars = attestation_utils.fetch_attestation_env_vars()
```

Comment:
I suggest renaming `env_vars` to `attestation_env`

Reply:
done